purchases/src/main/kotlin/com/revenuecat/purchases/CustomerInfo.kt

```diff
@@ -50,6 +52,9 @@ data class CustomerInfo(
     val originalAppUserId: String,
     val managementURL: Uri?,
     val originalPurchaseDate: Date?,
+
+    @ExperimentalPreviewRevenueCatPurchasesAPI
+    val virtualCurrencies: Map<String, VirtualCurrencyInfo>,
```

Comment:
One of the problems with data classes is that this modifies the constructor + copy constructor for `CustomerInfo`, in case someone is using it... We could try to limit the breaking change by adding a deprecated constructor without the new param

Reply:
Great idea @tonidero, I've added a deprecated constructor without the new param in 112656360778ada938db03c71e9aa25147910a7f 👍